modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/configurators/TargetPlatformRootProjectConfigurator.java

```diff
@@ -132,12 +149,23 @@ private void _addDependenciesTargetPlatformDistro(
 
 				@Override
 				public void execute(DependencySet dependencySet) {
+					Matcher dxpMatcher = _dxpVersionPattern.matcher(
```

Comment:
Variable names should end with the data type name. In this method there's only one `Matcher`, so `dxpVersion` can be renamed to `matcher`.